Makefile

```diff
@@ -94,16 +94,14 @@ lint-docstrings:
 # Check only production code. Ignore other flake errors which are captured by `lint`
 # Diff of committed changes (shows only changes introduced by your branch
 ifneq ($(strip $(BRANCH)),)
-	git diff $(BRANCH)...HEAD -- rasa | poetry run flake8 --select D --diff
+	git diff $(BRANCH)...HEAD -- rasa | poetry run ruff check --select D --diff
 endif
 
 	# Diff of uncommitted changes for running locally
-	git diff HEAD -- rasa | poetry run flake8 --select D --diff
+	git diff HEAD -- rasa | poetry run ruff check --select D --diff
```

Comment:
this is not working as intended right now; need to debug it (it also errors when there are no files)